packages/manager/apps/dedicated/client/app/dedicated/server/details/translations/Messages_fr_FR.json

```
@@ -344,7 +344,7 @@
   "server_tab_USB_STORAGE_order_select_duration": "Sélectionnez la durée :",
   "server_configuration_monitoring_activated": "Activé",
   "server_configuration_monitoring_disabled": "Désactivé",
-  "server_configuration_distribution": "Système (OS)",
+  "server_configuration_distribution": "Dernier Système d'Exploitation (OS) installé par OVH",
```

better without capital letter no ?
```suggestion
  "server_configuration_distribution": "Dernier Système d'exploitation (OS) installé par OVH",
```